File: Dev/sprint1/autenticacao/src/main/java/com/orion/autenticacao/models/Onibus.java
package com.orion.autenticacao.models;

public class Onibus {

    private String numero;
    private Linha Linha;

    public Onibus(String numero, Linha linha) {
        this.numero = numero;
        Linha = linha;
    }

    public String getNumero() {
        return numero;
    }

    public void setNumero(String numero) {
        this.numero = numero;
    }

    public Linha getLinha() {
        return Linha;
    }

    public void setLinha(Linha linha) {
        Linha = linha;
    }
}

File: Dev/aplicativo-kotlin/app/src/main/java/com/orion/zenite/model/Onibus.kt
package com.orion.zenite.model

data class Onibus(
    val id: Int,
    val numero: String,
    val placa: String,
    val modelo: String,
    val fabricante: String,
    val acessivel: Boolean,
    val qtdPassageirosSentados: Int,
    val qtdPassageirosEmPe: Int,
    val gerente: Gerente,
    val conta: Conta,
    val linha: String,
    val motorista: String,
    val motoristaTelefone: String,
    val linhaId: Int
)
